app/(afterLogin)/(planningAndReporting)/planning-and-reporting/_components/reporting/index.tsx

```
@@ -216,13 +216,13 @@
     <Spin spinning={getReportLoading} tip="Loading...">
       <div className="min-h-screen">
         <div className="flex items-center my-4 gap-4">
-          {hasPermission && (
+          {/* {hasPermission && ( */}
             <EmployeeSearch
```

ESLint

[error] Delete `··`

(prettier/prettier)